frontend/src/App.js

```diff
@@ -1,111 +0,0 @@
-import React, { Component } from 'react';
-import axios from 'axios';
-import './App.css';
-import CreatureComponent from './components/CreatureComponent'
-
-// BASE URL
-const BASE_URL = "http://localhost:3005"
-
-class App extends Component {
-
-  constructor() {
-
-    super();
-
-    this.state = {
-
-      creatures: [],
-      newcreattext: '',
-      newcreatimg: '',
-      newcreatscary: ''
-
-    }
-
-  }
-
-  componentDidMount = () => {
-
-    axios.get(BASE_URL + "/api/creatures").then(res => {
-      this.setState({ creatures: res.data })
-    })
-
-  }
-
-  deletecreature = creatureID => {
-    // axios (DELETE)
-    axios.delete(BASE_URL + "/api/creatures/" + creatureID).then(res => {
-
-      this.setState({ creatures: res.data })
-
-    })
-  }
-
-  addcreature = creature => {
-    // axios (POST)
-    axios.post(BASE_URL + "/api/creatures", { newcreature: this.state.newcreattext, newimg: this.state.newcreatimg, scary: { level: this.state.newcreatscary } }).then(resp => {
-
```

Comment:
I would clean this up a little with some destructuring and spacing your object out onto separate lines.